lsfd-cmd/lsfd.c

```diff
@@ -84,6 +84,8 @@ static int kcmp(pid_t pid1 __attribute__((__unused__)),
 
 #include "lsfd.h"
 
+struct lsfd_control *lsfd;
+
```

Comment:
Do we need this?

I tried removing this global var.

https://github.com/masatake/util-linux/commit/c4ad65b9f651350c9cd718776fdb30aa0668f0c8

I have to change many places but the changes are very simple.